docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+
+Support for running on [Kubernetes](https://kubernetes.io/) is available in experimental status. The feature set is
+currently limited and not well-tested. This should not be used in production environments.
+
+## Setting Up Docker Images
+
+In order to run Spark on Kubernetes, a Docker image must be built and available on an accessible Docker registry. Spark
```

Comment:
There is no direct dependency on Docker, so, s/Docker image/container image/. We can say that in the common case, we need to build docker images and push it to their image registry.